src/main/java/co/novu/api/messages/MessageHandler.java

```diff
@@ -1,39 +1,41 @@
 package co.novu.api.messages;
 
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
 import co.novu.api.messages.requests.MessageRequest;
 import co.novu.api.messages.responses.DeleteMessageResponse;
 import co.novu.api.messages.responses.MessageResponse;
-import co.novu.common.base.NovuConfig;
+import co.novu.common.rest.NovuNetworkException;
 import co.novu.common.rest.RestHandler;
-import lombok.RequiredArgsConstructor;
+import retrofit2.Response;
 
-import java.util.HashMap;
-import java.util.Map;
-
-@RequiredArgsConstructor
 public class MessageHandler {
 
     private final RestHandler restHandler;
 
-    private final NovuConfig novuConfig;
+    private final MessageApi messageApi;
 
-    private static final String ENDPOINT = "messages";
+    public MessageHandler(RestHandler restHandler) {
```

Comment:
You'd need to modify the constructor of MessageHandler in Novu.java class. Please check that